doc/autoscale/experiment/README.md

```diff
@@ -1,26 +1,40 @@
 # Auto-scaling Experiment Design
 
+## Purpose
 
-## Environment Enviroment
+To verify effectiveness of PaddlePaddle's fault-tolerance and auto-scaling mechanism.
 
-- Kubernetes v1.6 cluster with 133 nodes.
-- PaddleCloud with latest develop branch installed.
-- We will train the [recognize_digits](https://github.com/PaddlePaddle/cloud/tree/develop/demo/recognize_digits) model in the experiment.
+## Metrics
 
+How the effectiveness are measured.
 
-## Experiment Metric
+1. Cluster computing resource overall utilization.
+    - the higher the better.
+    - higher utilization means less resource are idle.
+1. Task average pending time.
+    - the less the better.
+    - the less pending time the earlier developers and researchers can start seeing the training cost curve, and the better they can verify the training algorithm effectiveness.
+    - This is a common pain point of researchers with internal cloud.
+1. Task average execution time.
+    - the less the better.
+    - average execution time is another way of measuring computing resource utilization. the less execution time, the higher overall utilization.
+    - average execution time is also the way of measuring effectiveness of fault-tolerance. If the fault-tolerance is not working properly, the training job will simply fail or finish with significantly longer duration.
+1. Quality of service with Hybrid cluster
```

Comment:
Hybrid don't need capitalization (hybrid), maybe general purpose cluster is easier to understand?

Reply:
corrected.